py3status/modules/sysdata.py

```diff
@@ -7,10 +7,12 @@
     format: output format string
         *(default '[\?color=cpu CPU: {cpu_usage}%], '
         '[\?color=mem Mem: {mem_used}/{mem_total} GB ({mem_used_percent}%)]')*
-    padding: length of space padding to use on the left
-        (default 0)
-    precision: precision of values
-        (default 2)
+    format_percent: format for the values in percent
+        (default "{value:.2f}")
+    format_temp: format for the temperature value
+        (default "{value:.2f}{unit}")
+    format_value: format for the flat values
```

Comment:
I think it might be good to call this `format_mem` because with #520 we will really want a memory specific one with `{units}` whereas the loads in #552 might want there own format

As an aside `{mem_free}` might be nice at some point too.